eZ/Publish/API/Repository/Exceptions/ContentFieldValidationException.php

```diff
@@ -13,7 +13,7 @@
 /**
  * This Exception is thrown on create or update content one or more given fields are not valid.
  */
-abstract class ContentFieldValidationException extends ForbiddenException
+abstract class ContentFieldValidationException extends ContentValidationException
```

Comment:
@andrerom We will then have to change/remove this doc line: ["@throws ContentValidationException if a required field is missing or is set to an empty value"](https://github.com/ezsystems/ezpublish-kernel/blob/9558023a2ab2ac2c26e25bd65207b556b8d6f71a/eZ/Publish/Core/Repository/ContentService.php#L474)

And this is what Bertrand said would require 7.0 tagged - but he may not have considered it a bug fix. (I agree that the wrong exception was used originally.)

If this is OK with you, we are go :)